app/assets/javascripts/discourse/app/components/composer-editor.gjs

```diff
@@ -941,6 +947,26 @@ export default class ComposerEditor extends Component {
     this._selectedFormTemplateId = value;
   }
 
+  @action
+  async updatePreviewFromForm() {
```

Comment:
nitpick: `updatePreviewFromForm` is a bit tongue twisting, how about simplifying it to `updateFormPreview`